src/systemd/sd-bus-vtable.h

```diff
@@ -90,7 +92,64 @@ struct sd_bus_vtable {
                 },                                                      \
         }
 
```

Comment:
I propose to add a version field to the table, filled automatically in the _SD_BUS_VTABLE_START element, with a number corresponding to the release.
And this because inferring the elements from the size is risky: if tomorrow a const char *newfield is added to the struct signal, this won't increase the sizeof(struct sd_bus_vtable), so it is not possible to make a distinction with the field present or absent

Reply:
we usually avoid explicit version check, since they are problematic in a world where downstream distros regularly backport stuff. Doing a feature-bitmask logic otoh would certainly make sense however. i.e. add a features field, and setting a specific bit to 1 means that some feature is now enabled.